.github/workflows/test-many-configs.yml

```diff
@@ -0,0 +1,42 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
Probably want to strip out these comments leftover from the initial tutoria/template.